mlflow/gateway/config.py

```diff
@@ -23,6 +23,7 @@ class Provider(str, Enum):
     # Note: Databricks Model Serving is only supported on Databricks
```

Comment:
Can we add docs and examples for this too?

Reply:
I filed a followup ticket for adding examples and docs for both OSS and Databricks Model Serving. I'll tackle it after the PR to add Databricks Model serving

Reply:
Since Databricks model serving isn't a release blocker, can we prioritize the OSS models docs first? We can deploy the OSS models docs after the 2.6 release if we don't have time to get them done :)